src/components/Dialog.tsx

```diff
@@ -320,7 +323,8 @@ export default function Dialog({ color, children, onClose, forceContain }: Dialo
           return (mountPoint?.childElementCount ?? 0) > 1 ? SlideAnimationType.PAGING : SlideAnimationType.CLOSING
       }
     },
-    modal
+    modal,
+    context.options?.animationType === DialogAnimationType.NONE
```

Comment:
nit: would prefer not doing the boolean evaluation here and instead assign it to a variable and pass that it

Reply:
done